lib/gui/index.html

```diff
@@ -54,10 +54,11 @@
     <div class="section-loader"
       ng-controller="StateController as state"
       ng-class="{
-        isFinish: state.is('success')
+        isFinish: state.currentName === 'success'
       }">
       <safe-webview
-        src="'https://etcher.io/success-banner/'"></safe-webview>
+        src="'https://etcher.io/success-banner/'"
+        refresh-now="state.previousName === 'success'"></safe-webview>
```

Comment:
I made sure to check if the previous prop was `false` and the current one `true`, so that a refresh is only triggered when the change from `false` → `true` happens and not `true` → `true`. React will sometimes trigger `componentWillReceiveProps()` when it's not just a prop update, so that's also why I took this precaution.